articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Branding
+description: How to brand Auth0 items to reflect your brand and desired user experience
+toc: true
+topics:
+    - b2c
+    - ciam
+    - branding
+    - universal-login
+    - login-pages
+    - change-password-pages
+    - custom-domains
+    - error-pages
+contentType: concept
+useCase:
+    - user-logout
+---
+# Branding
+
+Auth0 can be customized with a look and feel that aligns with your organization's brand requirements and user expectations. The consistency in appearance improves your customer's experience, giving them peace of mind that the pages they're using can be trusted and are secure.
+
+Some of the items that you can brand include:
+
+* Login, Change Password, and Error pages
+* Custom domains
+* Email templates
+
+Auth0 also supports [internationalization (I18N)](/i18n) and localization (L10N), both of which are important if you work with an international clientele.
+
+## Design considerations
+
+For some users, the out-of-box experience -- perhaps with just a touch of alteration -- is more than sufficient. However, for those wanting more control over appearance and user experience, Auth0 provides extensive flexibility when it comes to customizing and configuring things like your [Login](#universal-login), [Change Password](#change-password), and [Error](#errors) pages, [Custom Domain](#custom-domain) and [Email Templates](#email-templates). This allows you to create the user experience look and feel you need.
+
+When considering the items you want to brand, as well as how best to brand these items, you'll want to consider the following.
+
+* Will you brand your login page?
+
+* Will you need to localize your login page?
+
+* Will you customize emails so that they're not just branded, but vary based on user preference?
+
+* How will users know that they're still on your domain when they see your login page?
+
+* What do you need to do to provide additional browser security (e.g., implement Extended Validation)?
+
+* How will you direct users in the event of errors?
+
+### Universal Login and Login Pages
+
+The [Universal Login](/universal-login) process is the recommended method for authenticating users, and it centers around the Login Page. You can customize the Login Page via the Dashboard to support your organization's branding requirements. You can make [simple changes](/universal-login#simple-customization) that include changes to the logo, primary color, or background color, or you can make more [advanced-level changes](/universal-login#advanced-customization) that require you to modify the script powering the page itself directly.
```

Comment:
By the time the B2C PR is merged this may change, as advanced cust. will have its own page when we start differentiating between classic and new UL...